modules/data_team_helper/php/NDB_Form_data_team_helper.class.inc

```diff
@@ -286,12 +346,16 @@ class NDB_Form_Data_Team_Helper extends NDB_Form
      * @param unknown_type $test_name   the value of test name
      * @param string       $visit_label the value of visit label
      * @param string       $candidate   the value of candidate
+     * @param null         $site        the value of site
+     * @param null         $project     the value of project
```

Comment:
if the type for the variables `$site` and `$project` are null, why are they included at all?